apps/mull-api/src/app/event/event.service.ts

```diff
@@ -33,34 +35,34 @@ export class EventService {
   getEventsCoHostedByUser(coHostId: number): Promise<Event[]> {
     return this.eventRepository
       .createQueryBuilder('event')
-      .leftJoin('event.coHosts', 'user')
       .leftJoinAndSelect('event.location', 'location')
-      .where('user.id = :userId', { userId: coHostId })
+      .leftJoinAndSelect('event.host', 'host')
+      .leftJoin('event.coHosts', 'coHost')
+      .where('coHost.id = :userId', { userId: coHostId })
       .getMany();
   }
 
   getEventsAttendedByUser(userId: number): Promise<Event[]> {
     return this.eventRepository
       .createQueryBuilder('event')
-      .leftJoin('event.participants', 'user')
       .leftJoinAndSelect('event.location', 'location')
+      .leftJoinAndSelect('event.host', 'host')
+      .leftJoin('event.participants', 'user')
       .where('user.id = :userId', { userId })
       .getMany();
   }
 
   async isUserAttendingEvent(eventId: number, userId: number): Promise<boolean> {
     const event = await this.eventRepository
       .createQueryBuilder('event')
-      .leftJoin('event.participants', 'user')
-      .leftJoin('event.coHosts', 'coHost')
-      .where(
-        new Brackets((qb) => {
-          qb.where('coHost.id = :userId', { userId }).orWhere('user.id = :userId', { userId });
-        })
-      )
+      .leftJoin('event.participants', 'participant', 'participant.id = :userId', { userId })
+      .leftJoin('event.coHosts', 'coHost', 'coHost.id = :userId', { userId })
```

Comment:
these conditions are unnecessary since you have it below
```suggestion
      .leftJoin('event.participants', 'participant')
      .leftJoin('event.coHosts', 'coHost')
```

Reply:
I see, tried doing inner joins and removing the where clauses but it didn't end up working, those condition are leftovers, will remove.